relays/client-substrate/src/error.rs

```diff
@@ -42,6 +42,19 @@ pub enum Error {
 	Custom(String),
 }
 
+impl std::error::Error for Error {
```

Comment:
Perhaps we can agree on using the `thiserror` / `anyhow` combo for all our errors and start migrate towards this.

Reply:
Yes, logged here https://github.com/paritytech/parity-bridges-common/issues/857